SPECS/k3s/k3s.spec

```diff
@@ -1,15 +1,16 @@
+%define        _version 1.32.4
 Name:          k3s
 Summary:       K3s - Lightweight Kubernetes
-Version:       1.32.4
-Release:       2%{?dist}
+Version:       %{_version}+k3s1
```

Comment:
The number in `k3s<number>` is not always 1. So, we should not use a macro to automatically append the `k3s1` postfix.